### ISSUE TITLE
Sample_id ERROR

### ISSUE THREAD

**Issue by Vsd1106** [nanoporetech/pod5-file-format#56]
I've been converting some single fast5 read files to multi read without any issues, but when I'm trying to convert them to pod5 I am met with this error:ERROR:pod5:Enqueueing exception: batch_11.fast5 'sample_id'.
The multi read fast5 files work fine as I basecalled them with dorado to see if the issue was when converting them to multi read.

**Comment by HalfPhoton** [comment]
@Vsd1106 , are you using `v0.2.3`?

**Comment by Vsd1106** [comment]
I was using the 0.2.2 version

**Comment by HalfPhoton** [comment]
This issue should be fixed in [v0.2.3](https://github.com/nanoporetech/pod5-file-format/releases/tag/0.2.3)

```text
[0.2.3] 2023-06-26
Added
>>> Default field values (empty string) when converting fast5 files with missing fields
```
Please upgrade your installation with:

```bash
pip install -U pod5==0.2.3
````
